### ISSUE TITLE
The Windows Installer Service could not be accessed #2

### ISSUE THREAD

**Issue by Gravityzwell** [sandboxie-plus/Sandboxie#35]
Anybody have any examples where the fix https://github.com/sandboxie-plus/Sandboxie/issues/1
works?

I'm seeing the same error. 1809 with Acrobat DC setup. 

I never thought to make a list of installers where I got this error, going back a while, so I don't have anything more to test with.

**Comment by Gravityzwell** [comment]
One thing I noticed:
sc query MSIServer
works on a system where the installer works, but hangs on 1809, both in a sandbox of course.  It should show state as stopped. This may be an easy test to see if the problem occurs.

**Comment by Gravityzwell** [comment]
Its not so simple I guess. Tested a few more MSI's. The OpenJDK msi works on Sandboxie 5.33.6 and 5.40.1. Peazip doesn't work on either. Still looking for an MSI that works on 5.40.1 but not 5.33.6.

**Comment by DavidXanatos** [comment]
try openoffice that works with the fix

**Comment by Gravityzwell** [comment]
openoffice stops for me at the end with service could not be accessed error, for both 5.33.6 and 5.40.1. I was used to seeing that error right off with Acrobat and Peazip.

Others I've tried that worked, at least to start. Did not go to the end with them as I had assumed I'd see an error right away, but OpenOffice showed me different and I'll test them all later.

LibreOffice
OpenJDK
PDF-XChange
Python27 (because it has an MSI)
Tortoise 193
Tortoise GIT
Games for Windows Live redistributable

Note this was just a random list of what I had handy that has an MSI.

**Comment by Gravityzwell** [comment]
One that immediately throws a Sanboxed program has crashed error both 5.33.6 and 5.40.1:
https://www.devexpress.com/Products/AJAX-Control-Toolkit/

My current workaround is to install in a Sandbox in a VM running Server 2016 then copy the sandbox back to my working system. Probably an older Win10 would be better, but it works for me. At least it did for Acrobat. Ajax Control Toolkit started but I couldn't finish it because I don't have Visual Studio installed in that VM.

**Comment by Gravityzwell** [comment]
Actually I never did get Ajax Control Toolkit running in Sandboxie, even on Win 2016. I ended up monitoring it with Total Uninstall. Not that I ever planned to run that in a sandbox, but I wanted to know what the installer was doing, and Sandboxie is my first choice for such a thing, since I can easily see new files and registry.

**Comment by DavidXanatos** [comment]
The MSI problem is not an easy one to fix so some patience will be needed.

I am currently working on improving sbies debugging capabilities and a own UI that will be much more convenient to extent that the old MFC based control app.

**Comment by Gravityzwell** [comment]
Let me add that I'm grateful you're working on this. I've been using wumgr since 2018 and grateful for that as well. It's a relief to me to see Sandboxie on github. Then even better to see competent guys tending to it. And to hear a certificate is in the works! Great news all around :)

**Comment by Gravityzwell** [comment]
Just noticed that Peazip works if I run the setup EXE. It throws an error if running the MSI via msiexec /i peazip.msi.
https://sourceforge.net/projects/peazip/files/7.3.0/

**Comment by DavidXanatos** [comment]
I found a workaround to that issue, you need to add AnonymousLogon=n to the sandboxie ini to the boxes where you want to use msi and it will work, but not using an anonymouse tocken may weaken the protection.

**Comment by DavidXanatos** [comment]
With 5.41.2 this issue should be fixed :D

**Comment by Gravityzwell** [comment]
Not sure if this comment gets noticed since issue is closed, but INCREDIBLE work!

I tested the MSI's I remembered failing, Acrobat, Peazip, Open Office. All now work. Note that Open Office started but then failed for me at the end (previously).

Note that I was confused by the anonymous setting. I expected to see a different name in Task Manager but didn't, until I realized I needed to add both:
AllowSandboxieLogon=y
AnonymousLogon=n

Then it was clearly working.  I only saw the first option in the release notes. Unbelievable stuff. Portable mode? Awesome!
